### ISSUE TITLE
MeshCtrl Error 

### ISSUE THREAD

**Issue by FrankSanabria** [Ylianst/MeshCentral#1127]
Hi,

When I use MeshCtrl (Web GUI nobody not have access to groups), list the device groups, and when I add a user to device group, this not work, this is te command and result:

```
root@----:/opt/meshcentral# node ./node_modules/meshcentral/meshctrl listdevicegroups --loginuser "user1" --loginpass "pass" 
id, name
"AykQH3lxs2feqW17T5La6Uk9f1wzFWZmv9EJcMjtRtVNZR@jtPQp83i9G$CX3yjZ", "Test"
"9M0J7sEokBxbR58Hk3DgMS1lM4N@hG59bByyh5bBjC3$NpCmmPy18wt2gb06AVvr", "Externos"

root@-----:/opt/meshcentral# node ./node_modules/meshcentral/meshctrl addusertodevicegroup --loginuser "user1" --loginpass "pass" --id 9M0J7sEokBxbR58Hk3DgMS1lM4N@hG59bByyh5bBjC3$NpCmmPy18wt2gb06AVvr --userid user2 --fullrights
**Unknown group**
```

That have you a Great Day,

**Comment by Ylianst** [comment]
You may need to add quotes to the id because it contains the $ character. This said, I just updated MeshCtrl to support --group [groupname] in addition to --id [groupid]. It will be in MeshCentral v0.5.1-k when I next publish it. Should make things easier. If the group name has a space in it, you will need to use quotes.

**Comment by Ylianst** [comment]
Published MeshCentral v0.5.1-k earlier. Should fix this.

**Comment by FrankSanabria** [comment]
Hi, yes it's worked, thanks
